### ISSUE TITLE
missing-data chart with full_width: true doesn't react properly on window resize

### ISSUE THREAD

**Issue by avasenin** [mozilla/metrics-graphics#507]
I use code snippet below with full_width: true. When I resize browser window then graph doesn't change their size.

Graphs with data work as expected. 

```
            MG.data_graphic({
                title: "Missing Data",
                description: "This is an example of a graphic whose data is currently missing. We've also set the error option, which appends an error icon to the title and logs an error to the browser's console.",
                error: 'This data is blocked by Lorem Ipsum. Get your **** together, Ipsum.',
                chart_type: 'missing-data',
                missing_text: 'This is an example of a missing chart',
                target: '#graph-section',
                full_width: true,
                height: 200
            });
```

**Comment by avasenin** [comment]
I've found the reason of this and opened a separate PR for this https://github.com/mozilla/metrics-graphics/pull/508
